### DISCUSSION TITLE
Is there any way to attach as "read only"?

### DISCUSSION THREAD

**Discussion by lyssieth** [zellij-org/zellij#2788] [Q&A]
Akin to the `tmux` equivalent of `tmux attach-session -f read-only` or even `tmux attach-session -r`.

I'm asking because I have some commands I'm running and want to monitor, but don't want to accidentally cancel/break anything by hitting a key.

**Comment by baccenfutter** [comment]
This would also be very nice to have for [Ziina](https://github.com/ziinaio/ziina), allowing hosts to invite read-only peers.
